publication-doi-commons/src/main/java/no/unit/nva/publication/doi/PublicationMapper.java

```diff
@@ -0,0 +1,159 @@
+package no.unit.nva.publication.doi;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.net.URI;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+import no.unit.nva.publication.doi.dto.Contributor;
+import no.unit.nva.publication.doi.dto.Publication;
+import no.unit.nva.publication.doi.dto.Publication.Builder;
+import no.unit.nva.publication.doi.dto.PublicationDate;
+import no.unit.nva.publication.doi.dto.PublicationType;
+import nva.commons.utils.JsonUtils;
+
+public class PublicationMapper {
+
+    public static final String DEFAULT_ROOT = "/dynamodb";
+    public static final String PUBLICATION_TYPE = "Publication";
+    public static final String ERROR_MUST_BE_PUBLICATION_TYPE = "Must be a dynamodb stream record of type Publication";
+    protected static final ObjectMapper objectMapper = JsonUtils.objectMapper;
+
+    private static final JsonPointer CONTRIBUTOR_ARP_ID_JSON_POINTER = JsonPointer.compile("/m/identity/m/arpId/s");
```

Comment:
Move JsoinPointers to their respective model classes to clean up this class?

Reply:
They got their own class now.